src/sentry/web/frontend/account_notification.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import
+
+import itertools
+
+from django.contrib import messages
+from django.core.context_processors import csrf
+from django.db import transaction
+from django.http import HttpResponseRedirect
+from django.views.decorators.cache import never_cache
+from django.views.decorators.csrf import csrf_protect
+from django.utils.decorators import method_decorator
+
+from sudo.decorators import sudo_required
+
+from sentry.models import (
+    Project, ProjectStatus
+)
+from sentry.plugins import plugins
+from sentry.web.forms.accounts import ProjectEmailOptionsForm, NotificationSettingsForm
+from sentry.web.decorators import login_required
+from sentry.web.frontend.base import BaseView
+from sentry.web.helpers import render_to_response
+from sentry.utils.auth import get_auth_providers
+from sentry.utils.safe import safe_execute
+
+
+class AccountNotificationView(BaseView):
+    def get_settings_form(self, request):
```

Comment:
Let's just make this an attribute:

```python
class AccountNotificationView(BaseView):
    notifications_settings_form = NotificationsSettingsForm

    def handle(self, request):
        settings_form = self.notifications_settings_form(request.user, request.POST or None)
```

Then a subclass only needs to set the property.